tests/mocked/test_client_customizations.py

```diff
@@ -17,15 +17,15 @@ def test_custom_headers():
     client = Client("", headers=custom_headers)
 
     # pylint: disable=protected-access
-    assert client._client._config.headers_policy.headers == custom_headers
+    assert client._config.headers_policy.headers == custom_headers
 
 
 def test_custom_timeout():
     timeout = 300
     client = Client("", timeout=timeout)
 
     # pylint: disable=protected-access
-    assert client._client._config.retry_policy.timeout == timeout
+    assert client._config.retry_policy.timeout == timeout
```

Comment:
Tests failed here with autorest update to`6.13.16`. This could also further justify a 1.0 release.

Reply:
Update: this is not considered a breaking change and will not warrant a 1.0 release